src/store/getters/state-getters.ts

```diff
@@ -310,7 +311,22 @@ export const getCertifyState = (state: StateIF): CertifyIF => {
   return state.stateModel.certifyState
 }
 
-/** The Completing Party's email address. */
-export const getCompletingPartyEmail = (state: StateIF): string => {
```

Comment:
I renamed this because, at this point in the code, this is just the user's email/name/etc. When we assign it, then it becomes the completing party's data.